docs/coding.rst

```diff
@@ -578,17 +578,99 @@ editors and available plugins.
 Working with Protocol
 ---------------------
 
-Bedrock uses the `Protocol Design System <https://protocol.mozilla.org/>`_ to quickly produce consistent, stable components.
+Bedrock uses the `Protocol Design System <https://protocol.mozilla.org/>`_ to quickly produce consistent, stable components. This involves two steps:
 
-When we find we are frequently re-using code for a particular Protocol component (i.e.
-`Split <https://protocol.mozilla.org/patterns/organisms/split.html>`_), we convert it to a
-`macro template <https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates
-/macros-protocol.html/>`_ with parameters for customization.
+1. Adding the `correct markup <#styles-and-components>`_ or importing the `appropriate macro <#macros>`_ to the page's HTML file.
```

Comment:
oh, nice! I didn't realize we could just use the anchors, I kept putting in an absolute path with the prod url - will switch to this going forward

Reply:
I don't blame you, I had to really do some googling to understand reStructuredText! It doesn't come as second-nature as mark-down